.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
General: Why file names are not in pascal case (is this the convention in python)

Reply:
love you too